eng/Version.Details.xml

```diff
@@ -1,90 +1,90 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.WindowsDesktop.App.Ref" Version="7.0.0-rc.1.22379.3" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.WindowsDesktop.App.Ref" Version="8.0.0-alpha.1.22416.7" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
-      <Sha>d5046afdd2d0e6234ae3fc5bd4ae878594079cc7</Sha>
+      <Sha>d84ac6d64ab97febc153dfa903b52a828f47f44d</Sha>
     </Dependency>
-    <Dependency Name="VS.Redist.Common.WindowsDesktop.SharedFramework.x64.7.0" Version="7.0.0-rc.1.22379.3" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="VS.Redist.Common.WindowsDesktop.SharedFramework.x64.7.0" Version="7.0.0-rc.1.22366.1" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
-      <Sha>d5046afdd2d0e6234ae3fc5bd4ae878594079cc7</Sha>
+      <Sha>72e09152a35ce78eb06abf2c8c3971ab43817069</Sha>
     </Dependency>
-    <Dependency Name="VS.Redist.Common.WindowsDesktop.TargetingPack.x64.7.0" Version="7.0.0-rc.1.22379.3" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="VS.Redist.Common.WindowsDesktop.TargetingPack.x64.7.0" Version="7.0.0-rc.1.22366.1" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
-      <Sha>d5046afdd2d0e6234ae3fc5bd4ae878594079cc7</Sha>
+      <Sha>72e09152a35ce78eb06abf2c8c3971ab43817069</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.WindowsDesktop.App.Runtime.win-x64" Version="7.0.0-rc.1.22379.3" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.WindowsDesktop.App.Runtime.win-x64" Version="8.0.0-alpha.1.22416.7" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
-      <Sha>d5046afdd2d0e6234ae3fc5bd4ae878594079cc7</Sha>
+      <Sha>d84ac6d64ab97febc153dfa903b52a828f47f44d</Sha>
     </Dependency>
-    <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.ObjWriter" Version="1.0.0-alpha.1.22364.1" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
+    <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.ObjWriter" Version="1.0.0-alpha.1.22411.1" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
       <Uri>https://github.com/dotnet/llvm-project</Uri>
-      <Sha>75e182c26dbf5d9f8ccca0ac68ee63b0bebb17b8</Sha>
+      <Sha>e73d65f0f80655b463162bd41a8365377ba6565d</Sha>
       <SourceBuildTarball RepoName="llvm-project" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="VS.Redist.Common.NetCore.SharedFramework.x64.7.0" Version="7.0.0-rc.1.22411.12" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="VS.Redist.Common.NetCore.SharedFramework.x64.7.0" Version="7.0.0-rc.1.22415.13" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
This dependency name needs to be updated to VS.Redist.Common.NetCore.SharedFramework.x64.8.0

Reply:
@wtgodbe , should package name suffix version number? Do we maintain multiple versions at the same time?

Reply:
Winforms doing similar change here: https://github.com/dotnet/winforms/pull/7664